frontend/Breadcrumb.js

```diff
@@ -163,7 +163,7 @@ function getBreadcrumbPath(store: Store): BreadcrumbPath {
 }
 
 module.exports = decorate({
-  listeners: () => ['breadcrumbHead', 'selected'],
+  listeners: () => ['breadcrumbHead', 'selected', 'hideSymbol', 'hideDisplayNamed'],
```

Comment:
Why is `Breadcrumb` listening for `hideDisplayNamed` and `hideSymbol` when it doesn't use them?